lib/ScrollStrategy/ScrollStrategy.js

```diff
@@ -807,6 +811,7 @@ var MoonScrollStrategy = module.exports = kind(
 		this.$.hColumn.addRemoveClass('v-scroll-enabled', enabled);
 		this.$.pageUpControl.spotlight = enabled && this.spotlightPagingControls;
 		this.$.pageDownControl.spotlight = enabled && this.spotlightPagingControls;
+		this.set('vEnabled', enabled);
```

Comment:
I thought this same thing, but after a search discovered that `ScrollControls.js:updateScrollability` uses this property.
Does seem like a heavy approach...